### ISSUE TITLE
Add a version/sha footer to the blacklight app

### ISSUE THREAD

**Issue by mark-dce** [yalelibrary/YUL-DC#113]
**ACCEPTANCE**
- [X] ~The application footer displays the deployment date~ Split out into #139
- [X] The application footer displays the release or branch tag
- [X] I can get the git SHA for the deployed version 

**BREADCRUMBS**
https://curationexperts.github.io/playbook/every_project/git_sha.html
_NOTE: these instructions assume the application is deployed using Capistrano, we'll need to figure out how to extract the branch & SHA information from a container._

Timebox to a day.

**Comment by tilthouse** [comment]
This resolves the SHA and the branch name. For deployment timestamp, we will need to add something to the deployment pipeline.

**Comment by K8Sewell** [comment]
Screen shots of SHA and Branch in development.

![Screen Shot 2020-05-12 at 1.59.25 PM.png](https://images.zenhubusercontent.com/5eac63d9b58bd7046578e120/802b73a0-9b4a-4b57-ae3e-cf33bfd48533)

![Screen Shot 2020-05-12 at 1.59.39 PM.png](https://images.zenhubusercontent.com/5eac63d9b58bd7046578e120/33fae3e6-fd89-489e-ac7a-2a011716d9ea)

**Comment by mark-dce** [comment]
I wonder if we could drop something simple in the compose script that would drop a timestamp somewhere on the container filesystem when compose is run?  @mlooney or @geekscruff Thoughts?

**Comment by tilthouse** [comment]
It needs to happen when the container is deployed – if that's always the same thing, then sure. However, I tended to think it should come from the deploy system (Circle CI?), because that's the source of truth for deployments.

**Comment by geekscruff** [comment]
Could it be as simple as an ENV VAR we set at deploy time in Circle CI?

**Comment by mark-dce** [comment]
Hey - at standup today we agreed that we'd split out the "add a timestamp" requirement, since it seems to be hard and we have the tag and SHA done.  I'm going to add Julie's suggestion to ticket #139 - I'd suggested writing the timestamp to a file, but I'm fine with an ENV variable too.

**Comment by mark-dce** [comment]
Sending this back to In-Progress because we need the footer to show up in production-like cluster environments.

**Comment by K8Sewell** [comment]
https://github.com/yalelibrary/yul-dc-blacklight/pull/42

https://github.com/yalelibrary/yul-dc-blacklight/pull/43

**Comment by K8Sewell** [comment]
https://github.com/yalelibrary/yul-dc-blacklight/pull/43

Tests added for content check of footer and PR is ready for review.

NOTE: To pass rubocop checks an adjustment was made to the rubocop configuration to not require that the describe statement contain a class or module when testing anything in a views folder.

**Comment by tilthouse** [comment]
To elaborate on the rubocop changes: 
There is a cop that requires a class or method on `describe` statements in specs. This does not make sense for view specs (wether or not view specs themselves make sense is a separate issue). So we disabled the cop for those specs.

Seems like "temporarily" disabled cops are going in a "todo" config file, so we removed an example of this that covered by the overall rule change.
